recipes/s2n/all/conanfile.py

```diff
@@ -0,0 +1,77 @@
+import os
+from conans import ConanFile, CMake, tools
```

Comment:
Just to be conform to all recipes.
```suggestion
from conans import ConanFile, CMake, tools
import os
```